src/main/resources/application-dev.yml

```diff
@@ -18,3 +18,17 @@ jobs:
   remove-aed-final-retention-period-data:
     name: "RemoveAedFinalRetentionPeriodData"
     cron: "0 5/10 * ? * *" # every 10 minutes starting at :05
+
+email:
+  client:
+    sender: noreply_dev_erouser@erop.ierds.uk
+    allow-list-enabled: true
+    allow-list-domains:
+      - valtech.com
+      - levellingup.gov.uk
+      - softwire.com
+  content:
+    portal-base-url: https://ero.dev.erop.ierds.uk
```

Comment:
Should this `portal-base-url` not be derived from infra `env` variable as infra is more closer to these domains?

Reply:
Thanks @vishalgupta-valtech for your approval on https://github.com/cabinetoffice/eip-ero-infra/pull/756 that has now been merged